### PR TITLE
first release

### PR DESCRIPTION
**Pull request by Minhaajudeen**

### CODE REVIEW

File: client/src/components/staff-portal/attendance-maintain.js [hunk 1]
```diff
@@ -375,9 +375,10 @@ class AttendanceMaintain extends Component {
 
   applyStudentLeave() {
     let isUpdated = false;
-    // console.log("userid0", this.state.selectedStudent, this.state.selectedSemester, this.state.userID, this.state.description)
+
     const formData = new FormData();
-    formData.append("userID", this.state.selectedStudent.toUpperCase());
+    // formData.append("userID", this.state.selectedStudent.toUpperCase());
+    formData.append("userID", "13IT206");
```
Comment (AravindKumaran) [inline client/src/components/staff-portal/attendance-maintain.js:381]: user id is hard coded here?

File: models/db.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
```
Comment (AravindKumaran) [inline models/db.config.js:1]: why models folder is having this config file? we already have a config file right?

File: client/src/components/staff-portal/attendance-maintain.js (hunk 1 above)
Comment (dev-Rasla) [reply client/src/components/staff-portal/attendance-maintain.js:381]: I forget to remove it! it's for testing! Sorry! Now its removed

File: models/db.config.js (hunk 2 above)
Comment (dev-Rasla) [reply models/db.config.js:1]: It's for Avoid Code Repetition! Niraya place la mongodb connection theva pathuchi! so db connection code thani file la potu theva ulla edathula import pannikitan! main sever file knjam clean aa irukum!

**Pull request merged**